Changelog.md

```diff
@@ -38,6 +38,8 @@
   ```xml
   unwanted text<struct>...</struct>
   ```
+- [#523]: Fix incorrect handling of `xs:list`s with encoded spaces: their are still
```

Comment:
Did you mean "they still act as delimiters"?

Reply:
Yes